tests/mock_tables/state_db.json

```diff
@@ -94,6 +94,10 @@
     "position_in_parent": 1,
     "parent_name": "chassis 1"
   },
+  "PHYSICAL_ENTITY_INFO|psu_2_fan_1": {
```

Comment:
can you add the same change in tests/mock_tables/global_db - which is the information in host namespace in mulit-asic platform.

Reply:
added.